mbus-api/src/message.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
```

Comment:
will all of these be changed to protobuf later?

Reply:
If the messages in the buffer are protobuff, these might be changed.